.travis.yml

```diff
@@ -1,6 +1,6 @@
 # This config file for Travis CI utilizes https://github.com/coincar-sim/mrt_cmake_modules_ci/ package.
 sudo: required
-dist: xenial
+dist: bionic
```

Comment:
Is travis' bionic stable these days? It used to be a good idea to always wait for quite some time. That's why I only "recently" switched from trusty (https://github.com/coincar-sim/mrt_cmake_modules_ci/commit/08117b35f99bd530858ea55074f05bbfbdf30af9)

Reply:
Ah, I was already wondering why this env flag had no effect. But you're right, a docker container is run within the TravisCI container, one with [xenial 16.04](https://travis-ci.com/github/coincar-sim/lanelet_rviz_plugin_ros/jobs/508210029#L207) and one with [bionic 18.04](https://travis-ci.com/github/coincar-sim/lanelet_rviz_plugin_ros/jobs/508210030#L207).

Did you have issues with Travis bionic? I'd still assume that bionic should be better than EOL xenial ;)